api-specificatie/openapi.yaml

```diff
@@ -866,6 +866,8 @@ components:
     AdresHalCollectie:
       type: object
       properties:
+        _links:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalCollectionLinks'
```

Comment:
Moet dit niet HalPaginationLinks zijn? Als er bv. met een pand ID wordt gezocht naar adressen, dan kan de lijst van adressen dermate lang worden, dat paginering is vereist.

Reply:
Zal ik aanpassen. Wel nog even checken met de productowner.